Taggers/python/flashggDifferentialPhoIdInputsCorrection_cfi.py

```diff
@@ -2,7 +2,7 @@
 
 from RecoEgamma.EgammaTools.regressionModifier_cfi import regressionModifier 
 from flashgg.Taggers.globalVariables_cff import globalVariables
-
+from flashgg.Taggers.flashggTags_cff import *
```

Comment:
Seems unlikely that this is needed - please remove unless it really is

Reply:
Thanks Ed. I removed it now. . Initially I implemented training xml paths here, so I added that line. Now I defined training xml paths in flashggTagSequence_cfi.